js/main.js

```diff
@@ -247,23 +287,22 @@ function randomizeMissingLyrics(lyricCardNumber) {
 }
 
 function handleSubmitLyrics(event) {
-  var $cardLyrics = document.querySelectorAll('.lyrics');
-  var $p = $cardLyrics[data.lyricCard].querySelectorAll('p');
-  var $span = $p[data.randomLyricLine[data.lyricCard]].querySelectorAll('span');
+  const $cardLyrics = document.querySelectorAll('.lyrics');
+  const $p = $cardLyrics[data.lyricCard].querySelectorAll('p');
+  const $span = $p[data.randomLyricLine[data.lyricCard]].querySelectorAll('span');
   data.submittedWords.push($lyricsInput.value);
-  var $wordsOfInput = $lyricsInput.value.split(' ');
-  var numberOfMissingWords = 2;
-  for (var i = 0; i < numberOfMissingWords; i++) {
-    var strippedWords = data.missingWords[data.lyricCard][i].split('?').join('').split('!').join('').split(',').join('').split('.').join('').split('\'').join('').split('"').join('').split('(').join('').split(')').join('');
-    var noCaps = strippedWords.toLowerCase();
+  const $wordsOfInput = $lyricsInput.value.split(' ');
+  const numberOfMissingWords = 2;
+  for (let i = 0; i < numberOfMissingWords; i++) {
+    const strippedWords = data.missingWords[data.lyricCard][i].toLowerCase().split('?').join('').split('!').join('').split(',').join('').split('.').join('').split('\'').join('').split('"').join('').split('(').join('').split(')').join('');
```

Comment:
This should use a regular expression instead. Something like.

```js
const strippedWords = data.missingWords[data.lyricCard][i].toLowerCase().replace(/[?!,."\\()]/g, '')
```

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Regular_Expressions/Groups_and_Ranges